### PR TITLE
cleanup html input docs

### PR DESCRIPTION
**Pull request by d3caf**
- General cleanup and editing
- Add types section
- Tweak example code
- Remove old YouTube link

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/html/attributes/input/index.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Input
 ---
-## Input
-The HTML `<input>` tag is used within a form to declare an input element.
-It allows the user to enter data.
+
+# Input
```
Comment (huyenltnguyen) [inline guide/english/html/attributes/input/index.md]: Please change the heading back to `##`. 
You can use this page for reference: https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english/css/borders/index.md. Click on the <kbd>Raw</kbd> button if you want to see its markdown.

File: guide/english/html/attributes/input/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+The HTML `<input>` tag is used to create an element which allows a user to input data. Typically this would be within a `<form>` element, although it's not required.
 
 ## Example
```
Comment (huyenltnguyen) [inline guide/english/html/attributes/input/index.md]: I know this is not your change, but while you are at it, I hope you don't mind changing this to `###`.

File: guide/english/html/attributes/input/index.md [hunk 3]
```diff
@@ -15,28 +15,28 @@ It allows the user to enter data.
...
-<a href="https://www.youtube.com/watch?v=qJ9ZkxmVf5s">Youtube</a>
-
+In the above example, there is a form with two text fields which ask the user to enter their first and last names according to the labels specified. `<input type="submit">` is another type of input which is used to submit the form's data to the URL defined in the form's `action` attribute.
 
-## Input
-The HTML `<input>` tag is of many types to enter data. Some of them are:
-Type:Text(This is the most common type which is used to create general textboxes)
-Type:Password(This type is used for creation of password feilds)
-Type:Hidden(This is a special type of Input that is not shown to the user but it is used for passing information from one page to another while using <a href> tag)
+## Types
```
Comment (huyenltnguyen) [inline guide/english/html/attributes/input/index.md]: Please change this to `###` as well.

File: guide/english/html/attributes/input/index.md [hunk 4]
```diff
@@ -15,28 +15,28 @@ It allows the user to enter data.
...
+There are many types of `<input>` tags which can be specified with the input's `type` attribute. A few common types are:
+- **Text** - Creates a simple text box
+- **Password** - Creates a text field but masks the characters typed into the input
+- **Checkbox**
+- **Radio**
```
Comment (huyenltnguyen) [inline guide/english/html/attributes/input/index.md]: Please add an explanation for the `Checkbox` and `Radio`.

**Review by huyenltnguyen — CHANGES_REQUESTED** [review]

**Comment by andrew-anguiano** [comment]
@huyenltnguyen made the changes you requested. Let me know if you need anything else!

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

@andrew-anguiano Thank you for the additional changes and for your PR! 

I will merge this as soon as the Travis build completes.

Congratulations on your first contribution to this repo ✨ 🎉 We look forward to your next!

Happy coding!

**Pull request merged**